flag.go

```diff
@@ -84,7 +84,7 @@ func (fl *flag) init(clr color.Color, dontSetValue bool) error {
 			}
 		}
 	}
-	if !dontSetValue && fl.tag.dft != "" && dft != "" {
+	if !dontSetValue && fl.tag.dft != "" && dft != "" && fl.tag.injectDft {
```

Comment:
It can be safer if the flag will be negative (`ignore_dft` etc...), and if true - ignore.

Reply:
It is better practice to use positive for defaults